tests/plugins/tabletools/tabletools.js

```diff
@@ -192,6 +192,15 @@
 			} );
 		},
 
+		'test background color extraction': function() {
+			var bot = this.editorBot;
+
+			bender.tools.testInputOut( 'background-extraction', function( source, expected ) {
+				bot.setHtmlWithSelection( source );
+				assert.beautified.html( expected, bot.getData( true ) );
```

Comment:
This test fails in IE11 due to different order of rules in `background` style:
<img width="1001" alt="screen shot 2017-04-18 at 14 03 58" src="https://cloud.githubusercontent.com/assets/1061942/25129884/43d3297c-2440-11e7-9244-d48cae00b107.png">